x-pack/plugins/apm/public/components/app/Main/routeConfig.tsx

```diff
@@ -15,7 +15,7 @@ import { ServiceDetails } from '../ServiceDetails';
 import { TransactionDetails } from '../TransactionDetails';
 import { Home } from './Home';
 
-interface BreadcrumbArgs {
+export interface BreadcrumbProps {
```

Comment:
Would prefer to move these breadcrumb types/interfaces into the UpdateBreadcrumbs file, but then both of these files would need to import each other. I could move the types into a 3rd file, but we don't usually do that.

Thoughts on what's best here?

Reply:
As noted above, I'd like to see what the official types from `react-router-breadcrumbs-hoc` can give us.